### PR TITLE
Replaced all instances of IMEventSource.Logger.WriteInformation with LogHelper.LogInformation

### LINKED ISSUE #817: Ensure that all calls to the EventSource use LogHelper
Some parts of the code are calling: IdentityModelEventSource.Logger.WriteInformation
Replace this with the appropriate LogHelper function.

### PR DESCRIPTION
**Pull request by mafurman**
Fixes #817.

### CODE REVIEW

File: src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMetadataSerializer.cs [hunk 1]
```diff
@@ -28,6 +28,7 @@
 using System;
 using System.Security.Cryptography.X509Certificates;
 using System.Xml;
+using Microsoft.IdentityModel.Logging;
 using Microsoft.IdentityModel.Tokens;
 using Microsoft.IdentityModel.Xml;
 using static Microsoft.IdentityModel.Logging.IdentityModelEventSource;
```
Comment (brentschmaltz) [inline src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMetadataSerializer.cs]: Check if you still need any of these 'using static'

File: src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMessage.cs [hunk 2]
```diff
@@ -100,7 +101,7 @@ public WsFederationMessage(WsFederationMessage wsFederationMessage)
         {
             if (wsFederationMessage == null)
             {
-                Logger.WriteWarning(FormatInvariant(LogMessages.IDX22000, "wsfederationMessage"));
```
Comment (brentschmaltz) [inline src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMessage.cs]: Search for 'Logger.Write...' then

Comment (mafurman) [reply src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMessage.cs]: Done! All of those should be replaced now.

File: src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMetadataSerializer.cs (hunk 1 above)
Comment (mafurman) [reply src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMetadataSerializer.cs]: Got rid of all unnecessary 'using' statements in the files that were modified.

**Pull request merged**